packages/next-app/src/app/daoadmin/(components)/Sidebar.tsx

```diff
@@ -82,11 +66,26 @@ export function Sidebar() {
                 />
               </button>
             </div>
-            <FilterDropdown
-              filters={filters}
-              changeSelectedFilters={changeSelectedFilters}
-              clearSelectedFilter={clearSelectedFilter}
-            />
+            <Select
+              onValueChange={(value: string) => setSelectedFilter(value)}
+              theme={"light"}
+            >
+              {filters.map((filter) => (
+                <SelectPrimitive.Group key={filter.name}>
+                  <SelectPrimitive.Label className="py-1 px-2 text-sm text-gray-600">
+                    {filter.name}
+                  </SelectPrimitive.Label>
+                  <SelectItem value="" theme={"light"}>
```

Comment:
I don't have much context here but if the `filters` array has more than one element it will render another "no filter" right?